docs/content/contribution-guidelines/contributing-inc.md

```diff
@@ -33,6 +33,32 @@ Your assistance in improving documentation is highly valued, regardless of your
 
 To claim an issue that you are interested in, kindly leave a comment on the issue and request the maintainers to assign it to you.
 
+#### GitHub Slash Commands
+
+KubeStellar uses Prow and GitHub bots to help manage issues and pull requests through slash commands. These commands should be written as comments on their own line:
+
+**Issue Management Commands:**
+- `/assign @username` - Assign an issue to a specific user
+- `/unassign @username` - Remove assignment from a user  
+- `/assign` - Assign the issue to yourself
+- `/unassign` - Remove your assignment
+- `/good-first-issue` - Add the "good first issue" label
+- `/help-wanted` - Add the "help wanted" label
+- `/remove-good-first-issue` - Remove the "good first issue" label
+- `/remove-help-wanted` - Remove the "help wanted" label
```

Comment:
this command does not exist - please remove

Reply:
The doc at https://prow.k8s.io/command-help says that `/remove-help` _is_ a command. But testing shows that it provokes no reaction.

Reply:
we are not using prow for these commands. this is gha

Reply:
not looking to write a novel - just something quick for now - it can be embellished later.

Reply:
I do not think that telling the reader what tool is interpreting the slash commands and referring to that tool's documentation needs to take more than one sentence.

Reply:
I know that. what I am trying to avoid is more revisions that come after that. lets keep this simple and quick.